libcxx/test/std/ranges/range.adaptors/range.transform/adaptor.pass.cpp

```diff
@@ -20,11 +20,6 @@
 #include "test_macros.h"
```

Comment:
Do you need to include `"test_range.h"` here? Otherwise, this file is probably relying on a transitive include (unless `CanBePiped` is not used here?).

Reply:
Great eyes! In these cases, I was relying on the fact that the implementers of the adapters created their own header files (`types.h`) and those file includes the `test_range.h`. I have adjusted the patch to make direct inclusion of the `test_ranges.h` file but I am more than happy to go back to the old version if that is our standard.

Thanks for the sharp feedback!